setup.py

```diff
@@ -11,7 +11,13 @@
 import subprocess
 import sys
 import sysconfig
+from packaging import version
 from wheel.bdist_wheel import bdist_wheel
+
+# The MACOSX_DEPLOYMENT_TARGET for the library. If not set, it will use the
+# default value from python sysconfig.get_config_var('MACOSX_DEPLOYMENT_TARGET').
+MACOS_DEPLOYMENT_TARGET_OVERRIDE = "10.15"
```

Comment:
utterly trivial/style: move this variable declaration down, so it happen after all the `import` statements

currently, it's right before a weird conditional `import`

maybe move it down near the AWS_LIBS declaration